distributionviewer/core/static/js/app/components/views/example-date-range.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function(props) {
+  return (
+    <span className="date-range">
+      <time dateTime={props.from}>{props.from}</time> to <time dateTime={props.to}>{props.to}</time>
```

Comment:
Ah weird thought that was for reserved attribs only. Any way this works well just stood out a little bit while I was reading it.